airflow/operators/python.py

```diff
@@ -145,6 +150,7 @@ def __init__(
         op_kwargs: Optional[Dict] = None,
         templates_dict: Optional[Dict] = None,
         templates_exts: Optional[List[str]] = None,
+        show_return_value_in_logs: Optional[bool] = True,
```

Comment:
If the default is `True`, this shouldn't need to be `Optional`? What does passing in `None` mean?

Reply:
Oops!
That's correct, I was mistaken.
I'll remove the Optional.